packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 41.17.0 - January 15, 2018
+### Added
+#### salesforcedx-vscode
+
+* Support for use of Node 8([PR #256](https://github.com/forcedotcom/salesforcedx-vscode/pull/256))
```

Comment:
Does this change affect customers?

Reply:
@lcampos - We can probably leave this one out as well.